sys/cheri/cheric.h

```diff
@@ -328,7 +328,7 @@ __cheri_clear_low_ptr_bits(uintptr_t ptr, size_t bits_mask) {
  *
  */
 #define cheri_clear_low_ptr_bits(ptr, mask)                                    \
-  __cheri_clear_low_ptr_bits((uintptr_t)(ptr), __static_assert_sensible_low_bits(mask))
+  __cheri_clear_low_ptr_bits((uintptr_t)(ptr), __runtime_assert_sensible_low_bits(mask))
```

Comment:
This part isn't covered in the commit message.

Reply:
Will update it. IIRC the reason was introduced is that some of the clearing done in locking code in the purecap kernel uses a variable mask.

Reply:
I wonder whether we want to always allow non-static masks for all pointer bits macros at this point. @arichardson do you have an opinion on this?

Reply:
Allowing non-constant masks makes sense, I just didn't have a use-case for it when I initially added the macros. What we should really do is add them to the compiler...